frontend/app/src/components/form/bounty/index.tsx

```diff
@@ -444,8 +444,9 @@ function Form(props: FormProps) {
                           .map((item: FormField) => (
                             <Input
                               {...item}
+                              type={item.name === 'description' ? 'textarea' : item.type}
```

Comment:
instead of cheking `item.name === 'description'` twice lets define it as a boolean above and use it in the two places